tests/unit/scripts/test_tombstone.py

```diff
@@ -16,9 +16,25 @@ def test__tombstone_collection(self):
         self.assertFalse(c_v.canonical_collection.tombstoned)
         [self.assertFalse(d.canonical_dataset.tombstoned) for d in c_v.datasets]
 
+        context = MagicMock(spec=Context)
+        context.obj = {"business_logic": self.business_logic}
+        tombstone_collection(context, c_v.collection_id.id)
+
+        c_v = self.business_logic.get_collection_version(c_v.version_id, get_tombstoned=True)
+        self.assertTrue(c_v.canonical_collection.tombstoned)
+
+    def test__resurrect_collection(self):
+        c_v = self.generate_published_collection()
+        self.assertFalse(c_v.canonical_collection.tombstoned)
+        [self.assertFalse(d.canonical_dataset.tombstoned) for d in c_v.datasets]
```

Comment:
This and the above assertions are not needed. This is tested else were. This goes for the `test__tombstone_collection` as well.

Reply:
changed as requested